app/views/ChargeSummary.scala.html

```diff
@@ -57,7 +59,7 @@
 
 @dueDate = @{documentDetailWithDueDate.dueDate}
 
-@currentDate = @{LocalDate.now}
+@currentDate = @{dateService.getCurrentDate}
```

Comment:
again, no need to pass service in the view, pass the actual current date from the controller level